.ci/README.md

```diff
@@ -0,0 +1,18 @@
+# OpenMPI Continuous Integration (CI) Services
+## Mellanox OpenMPI CI
+[![Build Status](https://dev.azure.com/mlnx-swx/mellanox-ompi-ci-project/_apis/build/status/mellanox-ompi-ci-pipeline?branchName=master)](https://dev.azure.com/mlnx-swx/mellanox-ompi-ci-project/_build/latest?definitionId=6&branchName=master)
```

Comment:
There might be a different context: `mellanox-ompi-ci-project`, `mellanox-ompi-ci-pipeline` - I've intentionally make it so long to unambiguously understand what is this about. Otherwise there might be a mix of ambiguous `mellanox-ompi` entities.

Reply:
Don't get it. The project name will become the check name on github, right? The shorte the better IMO. And word "project" doesn't add anything to the meaning but eats space.

Reply:
There's a pipeline name in the GH PR status (for example, https://github.com/Mellanox/ompi/pull/7 - there's `mellanox-ompi-ci-pipeline` currently). But if you're within Azure DevOps interface and use something like `mellanox-ompi` or `mellanox-ompi-ci` - there may be several entities (e.g. project or a pipeline within the project) with such names. It isn't critical but let's discuss this verbally (I'll show what's the issue here).